css/wallscreens/buttons.css

```diff
@@ -1,16 +1,19 @@
 button, .button {
-  display: inline-block;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   background-color: var(--color-secondary);
   border: none;
   border-radius: 4px;
   box-shadow: 0px 4px 5px rgba(0,0,0,0.25);
   font-family: "Source Sans Pro", sans-sans-serif;;
-  font-size: 1em;
+  font-size: clamp(14px, 1.25vw, 1.25vw);
+  line-height: 1.5em;
   color: #000;
-  height: 40px;
-  line-height: 40px;
-  padding-left: 12px;
-  padding-right: 12px;
+  min-height:2vh;
```

Comment:
I think we should still set a min-height with a static number of 40px. Buttons smaller than 40px are going to be hard to press on phones or tablets. With this CSS, buttons have a height of 23px around 768px screenwidth. Do you want to 🍐 on this tomorrow?

Reply:
```suggestion
  min-height: clamp(40px, 2vh, 2vh);
```

🤷 ?

Reply:
Couldn't we also just use two fixed heights for buttons, one size for the physical screen and one for the web version (since we don't normally adjust button size when doing responsive stuff)? I like all the size adjustment stuff but think there will be some things, like buttons, that shouldn't have more than two, environment-based, size options.

Reply:
I guess it depends what our non-wallscreen design is; I thought 40px-tall buttons looked a little anemic at larger browser window sizes.